root/report/AnnotationsArtists.js

```diff
@@ -13,14 +13,13 @@ import useAnnotationColumns from './hooks/useAnnotationColumns.js';
 import {ANNOTATION_REPORT_TEXT} from './constants.js';
 import type {ReportArtistAnnotationT, ReportDataT} from './types.js';
 
-const AnnotationsArtists = ({
+component AnnotationsArtists(...{
   canBeFiltered,
   filtered,
   generated,
   items,
   pager,
-}: ReportDataT<ReportArtistAnnotationT>):
-React$Element<typeof ReportLayout> => {
+}: ReportDataT<ReportArtistAnnotationT>) {
```

Comment:
Why is there the three-dot ellipsis in the new syntax?
Can you please provide a reference in commit message to help with understanding?

Reply:
That's destructuring the props (https://flow.org/en/docs/react/component-syntax/#destructuring-parameters) so I can still use the existing types more easily rather than adding the typing specifically in the component declaration.

Reply:
Ok, added a bit of info to the commit message.